File: src/test/resources/typescript/comments-parsing/src/Comments.ts
/** Class doc */
export class Test {
  /** field doc */
  fieldVar: string;

  /**
   * method doc
   */
  test(/** param doc */ methodParam: string): void {}
}

export class NoComment {}

/** Interface doc */
export interface TestInterface {
  /** interface method doc */
  method(): void;
}

/** Enum doc */
export enum TestEnum {
  A
}

File: src/test/resources/python/comments-parsing/src/comments.py
class Test:
    """Class doc for Test."""

    field_var: str

    def test(self, method_param: str) -> str:
        """method doc for test."""
        return method_param


class NoComment:
    pass


class Factory:
    """Factory class doc."""

    @classmethod
    def build(cls, name: str) -> None:
        """classmethod doc for build."""
        _ = name
        return None
